Compile error: cannot convert istream to bool

When compiling with g++ 6.2:

```text
g++ -O2 -Wall -Wextra -DUSE_DICT_FILE -c -o zxcvbn-c++file.o zxcvbn.cpp
zxcvbn.cpp: In function ‘bool MyReadFile(FileHandle&, void*, unsigned int)’:
zxcvbn.cpp:215:35: error: cannot convert ‘std::basic_istream<char>::__istream_type {aka std::basic_istream<char>}’ to ‘bool’ in return
     return f.read((char *)Buf, Num);
                                   ^
```

--- a/zxcvbn.c
+++ b/zxcvbn.c
@@ -212,7 +212,7 @@ static inline void MyOpenFile(FileHandle & f, const char *Name)
 }
 static inline bool MyReadFile(FileHandle & f, void *Buf, unsigned int Num)
 {
-    return f.read((char *)Buf, Num);
+    return (bool)f.read((char *)Buf, Num);
 }
 static inline void MyCloseFile(FileHandle & f)
 {

